configs/nextflow.hopper.config

```diff
@@ -147,6 +148,7 @@ profiles {
     params.gatkreffolders = "${params.refpath}/gatk_cnv/cnvref/masked_hg38/gatk_ref"
     params.gens_accessdir = "/access/wgs/plot_data"
     params.accessdir = params.loaddir + "/wgs/"
+    params.svdprefix = "/fs1/resources/ref/hg38/VerifyBamID/resource/no_chr/1000g.phase3.100k.b38.vcf.gz.dat"
```

Comment:
Please update the variable name to make it clear what process it is intended for

I.e. `params.verifybamid2_svdprefix`

Otherwise it will be confusing when reading the config to know what this variable is intended for.